students/serek_wojciech/lesson_05_lists/swap_columns.py

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env python3
+"""Swap columns exercise"""
+
+
+def swap(matrix, col_1, col_2):
+    """Return chessboard fill sign for given field"""
+    for row in matrix:
+        row[col_1], row[col_2] = row[col_2], row[col_1]
+
+
+def main():
+    """Main function"""
+
+    rows, _ = [int(i) for i in input().split()]
+    matrix = [rows]
+    for _ in range(rows):
+        matrix.append([int(i) for i in input().split()])
+
+    col_1, col_2 = [int(i) for i in input().split()]
+
+    swap(matrix, col_1, col_2)
+
+    for row in matrix:
+        print(''.join(matrix[row]))
```

Comment:
Before join operation, you should cast each element of row list to string:
```print(' '.join([str(el) for el in row]))```

Reply:
Fixed.